generation/readme_gen/main.py

```
@@ -0,0 +1,365 @@
+    
+    return structure
+
+@register_forward("readme_gen")
+def forward(args):
+    """Main function to generate QA pairs with prompt as question and real README as answer"""
+    print("readme generation process begins!")
+    input_path = args['input_path']
+    output_path = args['output_path']
+    max_token = args.get('max_token')  # 获取max_token参数
+    
+    # 1. 加载仓库数据（包含真实README内容）
```

Ruff (0.12.2)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)